src/box/raft.c

```diff
@@ -45,6 +46,7 @@ struct raft box_raft_global = {
 };
 
 enum election_mode box_election_mode = ELECTION_MODE_INVALID;
+bool election_fencing_enabled = true;
```

Comment:
This variable can be made static. It is not used outside of this file.

Reply:
Fixed